packages/catalog-model/src/schema/kinds/API.v1alpha1.schema.json

```diff
@@ -46,7 +46,7 @@
             "type": {
               "type": "string",
               "description": "The type of the API definition.",
-              "examples": ["openapi", "asyncapi", "graphql", "grpc"],
+              "examples": ["openapi", "asyncapi", "graphql", "grpc", "trpc"],
```

Comment:
Technically this does affect the built package and calls for a changeset ... it may seem odd since it isn't exported, but we do compile it into ajv and I _think_ that it actually at runtime can make example values part of error responses and whatnot :)

Patch level is fine by me.

Reply:
Okay, I'll regenerate the changeset. This is my first time contributing to Backstage or a project which uses changesets, so it's all new to me.

Reply:
@freben I've updated the changeset for catalog-model to perform a minor version bump.